README.md

```diff
@@ -1383,6 +1385,28 @@ In order to monitor metrics (CPU, Memory, Disk usage...) about the VM during Cal
 
 The output of this script will be written to a `monitoring.log` file that will be available in the call gcs bucket when the call completes.  This feature is meant to run a script in the background during long-running processes.  It's possible that if the task is very short that the log file does not flush before de-localization happens and you will end up with a zero byte file.
 
+### Labelling Runs
+
+Every call to JES from a Cromwell instance will be automatically labelled by Cromwell so that it can be queried about later. The current label set automatically applied is:
+
+| Key | Value | Example | Notes |
+|-----|-------|---------|-------|
+| cromwell-root-workflow-id | The Cromwell ID given to the root workflow (i.e. the ID returned by Cromwell on submission) | cromwell-d4b412c5-bf3d-4169-91b0-1b635ce47a26 | To fit the required [format](#label-escaping-and-padding), we prefix with 'cromwell-' |
+| cromwell-root-workflow-name | The name of the root workflow | my-workflow | |
+| cromwell-workflow-name | The name of this job's workflow | my-sub-workflow | Different from cromwell-root-workflow-name if the task is called in a subworkflow |
+| wdl-task-name | The name of the WDL task of this job | my-task | |
+| wdl-call-name | The name of the WDL call of this job | my-call | Different from 'wdl-task-name' if it was called with an alias |
+
+#### Label Escaping and Padding
```

Comment:
(my last comment was lost in the github ether, so re-pinging @kcibul and/or @dvoet to give a thumbs up to this scheme (see it formatted [here](https://github.com/broadinstitute/cromwell/tree/cjl_workflow_id_label#label-escaping-and-padding)!)